### PR TITLE
## Pull Request: First Version of App - Featured/Setup1 Branch

### PR DESCRIPTION
**Pull request by aradradev**
### Description
This pull request presents the first version of the app, implemented on the featured/setup1 branch. The changes introduced in this branch align with the requirements specified for the project. The application has been created using Create React App and adheres to the best practices for JavaScript. Furthermore and the linter errors have been addressed.

### Changes Made
- Created a new branch called `featured/setup1` to contain the changes for the first version of the app.
- Set up the required linter configuration as per the instructions outlined in the linters-config README. All linter errors have been resolved.
- Followed the Gitflow methodology to ensure correct branching and merging strategies throughout the development process.
- Developed the React application using Create React App, leveraging the latest version of Node.js.
- Ensured that the application runs without any errors in the browser by thoroughly testing the functionality and addressing any issues encountered.

### Checklist
- [x] Set up linter configuration and resolve all linter errors.
- [x] Followed Gitflow methodology for branching and merging.
- [x] Created a React application using Create React App.
- [x] Ensured the application runs without errors in the browser.
- [x] Documented best practices for JavaScript followed in the project.

Please review the changes made in this pull request and provide any feedback or suggestions for improvement. Once approved, these changes can be merged into the develop branch.

### CODE REVIEW

**Review by Davidosky007 — COMMENTED** [review]
Hi @aradradev 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a descriptive readme , kindly Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './Index.css';
```
Comment (omarsalem7) [inline src/index.js]: - [ ] Kindly consider fixing the path styling path which starts with a small `i` not capital like this `./index.css` to make your app works properly

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @aradradev 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights

- Create react app properly ✔️
- The Pull Request has a proper title and description. ✔️
- Correct git-flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @aradradev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**